sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1458,7 +1477,7 @@ def task_worker(self):
                         # TODO: Make sure this doesn't impact other datapaths
                         api.set_lpmode(False)
                         self.port_dict[lport]['cmis_state'] = self.CMIS_STATE_AP_CONF
-                        self.port_dict[lport]['cmis_expired'] = now + datetime.timedelta(seconds=self.CMIS_DEF_EXPIRED)
+                        self.port_dict[lport]['cmis_expired'] = now + datetime.timedelta(seconds=self.get_cmis_dp_init_duration(api))
```

Comment:
this block is to handle CMIS DP_DEINIT state.
Instead of get_cmis_dp_init_duration(api) there should be function get_cmis_dp_deinit_duration(api) defined and invoked here?

DP_PATH_DEINIT_DURATION is defined at the bit range 4-8 vs DP_PATH_INIT_DURATION is defined at bit range 0-4 per CMIS spec (cmis.py), so values could be different.